.vimrc

```diff
@@ -404,7 +417,8 @@ autocmd VimEnter * if empty(expand('<amatch>'))|call FugitiveDetect(getcwd())|en
 " Signify
 nmap <leader>gj <plug>(signify-next-hunk)
 nmap <leader>gk <plug>(signify-prev-hunk)
-:nnoremap <Leader>gh :SignifyHunkDiff<CR>
+:nnoremap <Leader>ghd :SignifyHunkDiff<CR>
```

Comment:
Why was this changed?

Reply:
because I added a new map for `SignifyHunkUndo` so it not make sense only to have gh for `SignifyHunkDiff` now